docs/get-started/with-iot.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
what content will go here? suggest adding links to existing docs or blogs about IOT for now, or possibly removing the page until there is content for it. otherwise may give impression that it's not supported functionality.

Reply:
links added.